app/views/works/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<body>
```

Comment:
The body tag is in the layout file and there should only be one body in the HTML document. 

Perhaps you meant `main`